spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET #index' do
+    before(:each) { get users_path }
+    it 'is a success' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it "renders 'index' template" do
+      expect(response).to render_template(:index)
+    end
+
+    it 'should render the correct text in the index template' do
+      expect(response.body).to include('Here is the users/index<')
+    end
```

Comment:
you should ensure that the response body includes the correct placeholder text `expect(response.body).to include("Welcome to user index page")`